src/main/java/com/google/cloud/tools/gradle/appengine/base/extension/Deploy.java

```diff
@@ -15,7 +15,7 @@
  *
  */
 
-package com.google.cloud.tools.gradle.appengine.model;
+package com.google.cloud.tools.gradle.appengine.base.extension;
```

Comment:
It was a reflection of how gradle has some of their plugins (JavaBasePlugin), anyway, I changed it to core now.